src/styles/components.css

```
@@ -1,133 +1,366 @@
+  }
+  .history-toggle-btn:hover {
+    background-color: var(--neutral-400);
+  }
+  
+  .history-filter-select {
+    background-color: var(--neutral-200);
+    color: var(--text-primary);
+    border: 1px solid var(--neutral-400);
+  }
+  .history-filter-select:focus {
+    ring-color: var(--primary);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)